src/components/desktop/desktop-sub-header.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div
+    class="x-layout__sub-header x-layout-max-width-md desktop:x-layout-min-margin-32 large:x-layout-max-width-lg large:x-layout-min-margin-48"
+  >
+    <div class="x-layout__sub-header-content">
+      <DesktopSearchboxAlign class="x-layout-container">
+        <RelatedTagsFullPredictive />
```

Comment:
This way if we have to change the type of predictive we use we'll have to change it in two places, change the components structure so it's only one place (maybe with a prop to make things easier, something like [this flowchart](https://flowchart.fun/f#CIUwzg1gLg9gDgKAARNJWckaWEBjKASxgDtkUkAzAGxgEMiSBzJOAJ3gC4kAJEOgCYg2VWg0LNyFVOGjxe-ISIBiYxiwAKbEAMIFCANxBTpSAEoAVJAHc2dOHGEnTOWlCowRUABYhzVvBgAWzhSEBIoblV6dX8TSgBXamoAWmtCAR9WDjhuPkFhKiTqGwyfZxl0eXylJGVi0szvJC0dPSIjCpRLGzsHJxdpMDcPL19-JECQsIiohvSmuIQAXlXVhABvcgAiODo2XDZt7m2mPu8U3Gp8WCOAGh2fECCQY6RtoNIYMD28V4QAL4rNbLIA))